pkg/services/app/add.go

```diff
@@ -351,18 +351,25 @@ func (a *App) generateExternalRepoManifests(params AddParams, secretRef string,
 		return nil, nil, errors.Wrap(err, "could not generate target source manifests")
 	}
 
+	AppGoat, err := a.flux.CreateKustomization(params.Name, repoName, filepath.Join(".", "apps", params.Name), params.Namespace)
```

Comment:
I feel kind of dumb mentioning this, but this goes against naming conventions (capitalized local variable) :man_shrugging:

Reply:
that was an accident. It was named something else and i deleted the first part. thanks for catching that.